_sources/overview.rst.txt

```diff
@@ -0,0 +1,60 @@
+Overview
+=========
+
+SaltProc couples directly with the Monte Carlo neutron transport Serpent 2 and
```

Comment:
SaltProc couples directly with the Monte Carlo neutron transport **code** Serpent 2 and